clients/web/src/lib/api/queries/league/LeagueFixture.store.ts

```diff
@@ -11,19 +11,32 @@ export interface FixtureFranchise {
     };
 }
 
-export interface LeagueFixtureValue {
+export interface Fixture {
+    id: number;
     homeFranchise: FixtureFranchise;
     awayFranchise: FixtureFranchise;
 
     scheduleGroup: {description: string;};
     matches: Array<{
         id: number;
-        skillGroup: {description: string; ordinal: number;};
+        skillGroup: {
+            ordinal: number;
```

Comment:
Two ordinals?

Reply:
(check below, in profile)

Reply:
[a53d160](https://github.com/SprocketBot/sprocket/pull/309/commits/a53d16023a3fc9977307129a2065d693fbcb3ba0)